app/build.gradle

```diff
@@ -39,8 +39,10 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation 'com.squareup.okhttp3:logging-interceptor:4.5.0'
     implementation 'androidx.cardview:cardview:1.0.0'
+    implementation 'com.jakewharton:butterknife:10.2.1'
```

Comment:
![image](https://user-images.githubusercontent.com/26599673/80456146-6046ae80-892d-11ea-8db9-0314163d1dfb.png)

Reply:
Commit with this change removed